.github/workflows/test.yml

```diff
@@ -20,7 +20,7 @@ jobs:
   tests-fullSetup:
     strategy:
       matrix:
-        setup: [ {os: ubuntu-latest, profile: conan-linux-x86_64-release}, {os: windows-latest, profile: conan-windows-x86_64-release}, {os: macos-latest, profile: conan-macos-x86_64-release}] 
+        setup: [ {os: ubuntu-latest, profile: conan-linux-x86_64-release}, {os: ubuntu-20.04, profile: conan-linux-x86_64-release}, {os: windows-latest, profile: conan-windows-x86_64-release}, {os: macos-latest, profile: conan-macos-armv8-release}] 
```

Comment:
Should we include the ubuntu 20 tests? The packages are missing and because of that it takes a while (extra 7 minutes)

Reply:
Sure. Always nice to support older systems (clusterssometimes have older compilers)